### PR TITLE
Prepare release v0.6.1

### PR DESCRIPTION
**Pull request by alculquicondor**
Change-Id: I7cc659b2b013cd4a0b907b0e3002b877dac2e4ae

#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.6.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+## v0.6.1
+
+Changes Since `v0.6.0`:
+
+### Feature
+
+- Added MultiKueue  worker connection monitoring and reconnect. (#1809, @trasc)
```
Comment (tenzen-y) [inline CHANGELOG/CHANGELOG-0.6.md:7]:
```suggestion
- Added MultiKueue worker connection monitoring and reconnect. (#1809, @trasc)
```

nit. 
I have already applied this improvement to each PR.

Comment (alculquicondor) [reply CHANGELOG/CHANGELOG-0.6.md]: Done

**Review by tenzen-y — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**